os/net/mac/tsch/tsch-queue.c

```diff
@@ -421,8 +422,14 @@ tsch_queue_get_packet_for_nbr(const struct tsch_neighbor *n, struct tsch_link *l
 #if TSCH_WITH_LINK_SELECTOR
         int packet_attr_slotframe = queuebuf_attr(n->tx_array[get_index]->qb, PACKETBUF_ATTR_TSCH_SLOTFRAME);
         int packet_attr_timeslot = queuebuf_attr(n->tx_array[get_index]->qb, PACKETBUF_ATTR_TSCH_TIMESLOT);
-        if(packet_attr_slotframe != 0xffff && packet_attr_slotframe != link->slotframe_handle) {
-          return NULL;
+        if(packet_attr_slotframe != 0xffff){
+          if (packet_attr_slotframe != link->slotframe_handle) {
+            return NULL;
+          }
+          if(backup_link != NULL && packet_attr_slotframe != backup_link->slotframe_handle) {
```

Comment:
why not place this `if` in #if/#endif config condition? This allow easy backoff to standart behaviour?

also there is possible introduce new link option for "strong" rx-links, that are win vs tx?

Reply:
I would rather not have yet another configurable option.